src/dataflow/src/source/file.rs

```diff
@@ -340,45 +343,54 @@ pub fn read_file_task<Ctor, I, Out, Err>(
             //
             // https://github.com/notify-rs/notify/issues/240
             #[cfg(target_os = "macos")]
-            let (file_events_stream, handle) = {
```

Comment:
The `#[cfg(target_os = "macos")]` should change to "not Linux", right?

Reply:
Oy! Thanks. Fixed.